docs/kap_proposals/kap_7_remote_inventory.md

```diff
@@ -0,0 +1,64 @@
+# Remote Inventory Federation
+
+This feature would add the ability to Kapitan to fetch parts of the inventory from remote locations (https/git). This would allow users to combine different inventories from different sources and build modular infrastructure reusable across various repos.
+
+
+Author: @alpharoy14
+
+## Specification
+
+The configuration and declaration of remote inventories would be done in the hidden file `​.kapitan`.
+
+The file specifications are as follows:
+
+```yaml
+remoteInventory:
+    -   name: <inventory_name> #user defined
+        type: <inventory_type> #git\https
+        source: <source_of_inventory>
+        output_path: <relative_output_path>
+        ... type specific arguments
+```
+The command for adding remote inventories to the `.kapitan` file from cli will be:
```

Comment:
I think we can leave this feature behind for now. Maybe later we can revisit a more generic way of managing `.kapitan` with tooling. For now we expect users to manually edit the file.